Header header
int64 id
Quaternion quat_OBB
Point max_point_OBB
Point min_point_OBB
Point position_OBB


